sajibportfolio.css

```diff
@@ -1234,3 +1221,352 @@ button:disabled {
   color: #fff;
   background-color: #6070ff;
 }
+
+@media (min-width: 1024px) {
```

Comment:
- There is currently excess whitespace on the body of your desktop version when i scroll horizontally to the right direction. Can we fix this as well?

Here is what I mean 👇🏽 

![exps](https://user-images.githubusercontent.com/99246359/206776870-179ad1e6-de51-473f-91fd-737c8f04b249.PNG)

Reply:
<img width="884" alt="image" src="https://user-images.githubusercontent.com/39811497/206841779-8b3f7eb0-892e-421c-9c9f-a713e2543bef.png">

solved!